public/app/features/alerting/components/NewNotificationChannelForm.tsx

```diff
@@ -0,0 +1,150 @@
+          <Switch name="isDefault" ref={register} />
+        </Field>
+        <Field label="Include image" description="Captures an image and include it in the notification">
+          <Switch name="settings.uploadImage" ref={register} />
+        </Field>
+        {getValues().uploadImage && !imageRendererAvailable && (
+          <InfoBox title="No image renderer available/installed">
+            Grafana cannot find an image renderer to capture an image for the notification. Please make sure the Grafana
+            Image Renderer plugin is installed. Please contact your Grafana administrator to install the plugin.
+          </InfoBox>
+        )}
+        <Field
+          label="Disable Resolve Message"
+          description="Disable the resolve message [OK] that is sent when alerting state returns to false"
+        >
+          <Switch name="disableResolveMessage" ref={register} />
+        </Field>
+        <Field label="Send reminders" description="Send additional notifications for triggered alerts">
+          <Switch name="sendReminder" ref={register} />
+        </Field>
+        {getValues().sendReminder && (
+          <>
+            <Field
+              label="Send reminder every"
+              description="Specify how often reminders should be sent, e.g. every 30s, 1m, 10m, 30m or 1h etc."
+            >
+              <Input name="frequency" ref={register} />
+            </Field>
+            <InfoBox>
+              Alert reminders are sent after rules are evaluated. Therefore a reminder can never be sent more frequently
+              than a configured alert rule evaluation interval.
+            </InfoBox>
+          </>
+        )}
+      </div>
+      {selectedChannel && (
+        <>
+          <h3>{selectedChannel.heading}</h3>
+          {selectedChannel.info !== '' && <InfoBox>{selectedChannel.info}</InfoBox>}
+          {selectedChannel.options.map((option: Option, index: number) => {
+            const key = `${option.label}-${index}`;
+
+            // Some options can be dependent on other options, this determines what is selected in the dependency options
+            // I think this needs more thought.
+            const selectedOptionValue =
+              getValues()[`settings.${option.showWhen.field}`] &&
+              (getValues()[`settings.${option.showWhen.field}`] as SelectableValue<string>).value;
+
+            if (option.showWhen.field && selectedOptionValue !== option.showWhen.is) {
+              return null;
+            }
+
+            return (
+              <Field
+                key={key}
+                label={option.label}
+                description={option.description}
+                invalid={errors.settings && !!errors.settings[option.propertyName]}
+                error={
+                  errors.settings &&
+                  errors.settings[option.propertyName] &&
+                  errors.settings[option.propertyName].message
+                }
```

Comment:
```suggestion
                error={
                  errors.settings &&
                  errors.settings[option.propertyName]?.message
                }
```
nit: just ⅓ shorter :)